src/main/java/fr/jmmc/oiexplorer/core/gui/FitsImagePanel.java

```diff
@@ -206,13 +206,15 @@ private void initComponents() {
         jFormattedTextFieldRescaleFov = new javax.swing.JFormattedTextField();
         jLabelScale = new javax.swing.JLabel();
         jFormattedTextFieldScaleX = new javax.swing.JFormattedTextField();
+        jSliderFitsCube = new javax.swing.JSlider();
         jPanelOptions = new javax.swing.JPanel();
         jLabelLutTable = new javax.swing.JLabel();
         jComboBoxLUT = new javax.swing.JComboBox();
         jLabelColorScale = new javax.swing.JLabel();
         jComboBoxColorScale = new javax.swing.JComboBox();
         jButtonDisplayKeywords = new javax.swing.JButton();
         jToggleButtonRuler = new javax.swing.JToggleButton();
+        sliderPanel = new fr.jmmc.oiexplorer.core.gui.SliderPanel(this);
```

Comment:
No, the SliderPanel instance should be created and managed elsewhere (parent view...) to avoid any conflict with ASPRO2 (AnimatorPanel will be merged with SliderPanel later).